src/runtime/step_runner.js

```diff
@@ -8,7 +8,7 @@ const {beginTiming, endTiming} = Time
 
 async function run({attachmentManager, defaultTimeout, scenarioResult, step, stepDefinition, transformLookup, world}) {
   beginTiming()
-  const parameters = stepDefinition.getInvocationParameters({scenarioResult, step, transformLookup})
+  const parameters = await Promise.all(stepDefinition.getInvocationParameters({scenarioResult, step, transformLookup}))
```

Comment:
Alright sounds good. We need to test that if the async transformer rejects that the step gets reported as a failure. At the moment, I believe an async transformer rejecting might just kill the suite.